src/client/js/index.js

```diff
@@ -9,5 +9,6 @@ import './components/custom-select.js'
 import './nav.js'
 import './user-menu.js'
 import './partials/breaches.js'
+import './partials/email-preview.js'
```

Comment:
can we import this separately from the index bundle, so that non-dev users aren't affected?